endpoints/swagger_generator.py

```diff
@@ -547,12 +547,21 @@ def __definitions_descriptor(self):
     # Filter out any keys that aren't 'properties' or 'type'
     result = {}
     for def_key, def_value in self.__parser.schemas().iteritems():
-      prop_keys = def_value.keys()
-      if 'properties' in prop_keys or 'type' in prop_keys:
+      def_item_keys = def_value.keys()
+      if 'properties' in def_item_keys or 'type' in def_item_keys:
```

Comment:
Yep, I can change it to 'properties' in def_value, etc. Not sure about 'required' in prop_value, though. That's a dict in and of itself.